dev.tmpl.env

```diff
@@ -19,9 +19,4 @@ XOS_API_ENDPOINT=http://<your ip>:8000/api
 # XOS_AUTH_TOKEN token's user requires staff privileges for PATCH to /api/assets
 XOS_AUTH_TOKEN=AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
 
-# If not using Docker and SMB, set local directories to use
-# WATCH_FOLDER=/Users/stevejobs/acmi/devmount/watch
-# MASTER_FOLDER=/Users/stevejobs/acmi/devmount/master
-# ACCESS_FOLDER=/Users/stevejobs/acmi/devmount/access
-# WEB_FOLDER=/Users/stevejobs/acmi/devmount/web/
-# OUTPUT_FOLDER=/Users/stevejobs/acmi/devmount/output
+FLEXIBLE_MASTER_NAMING=False
```

Comment:
Add trailing slash to template: `XOS_API_ENDPOINT=http://<your ip>:8000/api/`

Reply:
Good call, done.